Documentation/subcommands/config.md

```diff
@@ -0,0 +1,130 @@
+# rkt config
+
+This command prints the rkt configuration per stage in JSON format.
+
+## Structure
+
+The general structure is a simple hierarchy consisting of the following top-level elements:
+
+```
+{
+	"stage0": [...],
+	"stage1": [...]
+}
+```
+
+The entries "stage0" and "stage1" refer to stage-specific configuration. The generated output are valid configuration entries as specified in the configuration documentation.
+
+The "stage0" entry contains entries of rktKind "auth", "dockerAuth", and "paths". Note that the `config` subcommand will output separate entries per "auth" domain and separate entries per "dockerAuth" registry. While it is possible to specify an array of strings in the input configuration rkt internally merges configuration state from different directories potentially creating multiple entries.
```

Comment:
I addressed this by adding a statement that the current merged/non-merged output behavior is subject to change.